src/manage/client-list.js

```diff
@@ -52,14 +77,13 @@ class ClientList extends React.Component {
               </StyledLogo>
```

Comment:
My only comment would be to have the avatar and the name | type on the same stack group. This way, we can control the separation better.